kubeflow/knative-build/knative-build.libsonnet

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
Is this converted from https://raw.githubusercontent.com/knative/serving/v0.1.1/third_party/config/build/release.yaml?
anything changed?